shell/edit/harvesterhci.io.setting/index.vue

```diff
@@ -159,6 +159,7 @@ export default {
           :localized-label="true"
           :mode="mode"
           :options="enumOptions"
+          @keydown.native.enter.prevent="()=>{}"
```

Comment:
We should try to solve this in a more general way so we don't accidentally cause this bug on other pages in the future. We can just put `@submit.prevent` on the `form` element here: https://github.com/rancher/dashboard/blob/master/shell/components/CruResource.vue#L344

This only happens with `<form>` that have only one input element, so I do not expect this change to cause problems elsewhere. https://html.spec.whatwg.org/multipage/form-control-infrastructure.html#implicit-submission

Reply:
Done, add the `@submit.prevent` for `CruResource` form.